db/db_compaction_test.cc

```diff
@@ -4616,7 +4617,6 @@ TEST_F(DBCompactionTest, LevelTtlCascadingCompactions) {
               ttl_compactions++;
             }
           });
-      ROCKSDB_NAMESPACE::SyncPoint::GetInstance()->EnableProcessing();
```

Comment:
Won't it be executed on Line 4690-4691:

```
      if (if_restart) {
        Reopen(options);
```

Reply:
Actually I see your point. "SanitizeOptions::AfterChangeMaxOpenFiles" looks like it's supposed to be executed irrespective of `if_restart`, so it's sometimes confusingly skipped.